usr/share/rear/layout/prepare/GNU/Linux/131_include_new_btrfs_subvolumes_code.sh

```diff
@@ -1,13 +1,16 @@
 
```

Comment:
On this occation, the file could be renamed to something like `131_include_btrfs_subvolumes_generic_code.sh`.

Reply:
Done via
https://github.com/rear/rear/pull/2080/commits/cd2bd2e60a6824b2244ef91abb34f8d239dcb500